src/app/layout/navbar/navbar.component.html

```diff
@@ -2,4 +2,6 @@
   <div class="container">
     <a class="navbar-brand" href="#">GP4Tech</a>
   </div>
+  <a class="" (click)="switchLanguage('en')">{{ 'language.english' | translate }}</a> |
```

Comment:
If class attribute is not used should be removed.

But in this case i think we should add a class or something to denote it as selected? for now you can use a dummy class that in another PR can have the styles that we are going to apply with materialize.